### PR TITLE
Added use examples of context timeout to API v6

### PR DESCRIPTION
**Pull request by Opelord**
Issue: KNTK-330
This PR is based on PR #54

### CODE REVIEW

**Review by mmac-m3a — APPROVED** [review]
LGTM

**Comment by mmac-m3a** [comment]
Waiting for @Fenthick to approve before merging.

**Comment by Fenthick** [comment]
@Opelord I see that it has no common code with API v5 timeouts, so it can be rebased onto master, to simplify the review process.

File: apiv6/README.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 This module implements Go client for Kentik API v6.
 
+## Usage
+
+To configure timeout for API client use context.WithTimeout() and pass it to the request function.
```
Comment (Fenthick) [inline apiv6/README.md:7]:
```suggestion
To configure timeout for a client call use _context.WithTimeout()_ and pass it to the request function.
```
... not to suggest that it modifies the API client.

**Review by Fenthick — COMMENTED** [review]
Please rebase onto master and accept one minor suggestion.

File: apiv6/README.md (hunk 1 above)
Comment (Fenthick) [reply apiv6/README.md]: Done

**Review by Fenthick — APPROVED** [review]

**Review by mmac-m3a — APPROVED** [review]

**Pull request merged**